prow/README.md

```diff
@@ -1,71 +1,159 @@
 # Prow
 
-Prow dashboard: http://prow.apps.ci.metal3.io
+Metal3 Prow dashboard: https://prow.apps.test.metal3.io
 
 ## Access Controls
 
 * To merge, patches must have `/approve` and `/lgtm`, which apply the `approved` and `lgtm` labels
 
-* Only members of the `metal3-io` github org can use `/lgtm`.
-
-* The use of `/approve` is controlled by the `OWNERS` file in each repository.
+* The use of `/approve` and `/lgtm` is controlled by the `OWNERS` file in each repository.
   See the [OWNERS spec](https://go.k8s.io/owners) for more details about how
   to manage access to all or part of a repo with this file.
 
 * Tests will run automatically for PRs authored by **public** members of the
   `metal3-io` github organization.  Members of the github org can run
   `/ok-to-test` for PRs authored by those not in the github org.
 
-See the [Prow command help](https://prow.apps.ci.metal3.io/command-help) for
+See the [Prow command help](https://prow.apps.test.metal3.io/command-help) for
 more information about who can run each prow command.
 
-## Prow Setup
+## GCS bucket
+
+Google Cloud Storage (GCS) is required to store job artifacts. Follow the docs to set
+up GCS and create a bucket with enough permissions.
 
-For more info about the CI cluster, see
-[project-infra/clusters/ci/README.md](../clusters/ci/README.md).
+1. Create a [bucket](https://cloud.google.com/storage/docs/creating-buckets)
+1. Create a [service account](https://cloud.google.com/iam/docs/creating-managing-service-accounts)
+1. On the created bucket, grant `storage.objects.create` access for the service account and `storage.objectViewer`
+   permission to `allUser`.
+
+## Setup
 
 Prow was set up by following these instructions: https://github.com/kubernetes/test-infra/blob/master/prow/getting_started_deploy.md
 
-```
-go get -u k8s.io/test-infra/prow/cmd/tackle
-tackle
-```
+1. Create DNS record(s) to point to your cluster.
+
+1. Prow components are running on `default` namespace, while `test-pods` namespace
+   is used by pods running actual CI jobs.
+
+1. [ghProxy](https://github.com/kubernetes/test-infra/tree/master/ghproxy) is a reverse proxy,
+   and is meant to reduce API token usage. We need persistent volume for ghProxy, and for that
+   purpose we rely on `hostPath` type of `PersistentVolume` in our cluster due to the limitation
+   in the underlying infrastructure. Ideally, you should use a dynamic storage provisioner.
+
+   Create a /mnt/prow directory to store the persistent volume data:
+
+     ```shell
+     $ sudo mkdir /mnt/prow
+     ```
+
+     Create the PersistentVolume:
+
+     ```shell
+     $ kubectl apply -f persistent_volume.yaml
+     ```
+
+1. Grant a user `cluster-admin` role in all namespaces to create cluster resources.
+
+    ```shell
+    $ kubectl create clusterrolebinding cluster-admin-binding-"${USER}" \
+      --clusterrole=cluster-admin --user="${USER}"
+    ```
```

Comment:
```suggestion
    ```shell
    USER=cluster-admin
    $ kubectl create clusterrolebinding cluster-admin-binding-"${USER}" \
      --clusterrole=cluster-admin --user="${USER}"
    ```
```

Reply:
I think this is correct, because we grant `--clusterrole=cluster-admin` permission to  ${USER}, any ${USER} can be different depending on where the cluster is running.

Reply:
I think that user refers to the `service account`. 
If the OS user and service account are the same, then it is ok.

Reply:
I followed instructions here https://github.com/kubernetes/test-infra/blob/master/prow/getting_started_deploy.md#create-cluster-role-bindings.